.github/workflows/mlflow-integration-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: MLflow-UC OSS Integration Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
Should there be any path restrictions here, or do you want to ensure that this suite runs for any changes in the repo?

Reply:
I think it should run for every PR.  Changes to ACLs, underlying database connectivity, etc could all impact the integration.  I'd end up covering most of the codebase if I pattern matched against everything that could impact the integration.